book/website/community-handbook/governance/gov-roles.md

```diff
@@ -0,0 +1,239 @@
+
+Please capture these decisions in the working group's charter.
+
+## Secretary
+
+> This text is adapted from the "What is the role of the Secretary" documentation in the [DIY Committee Guide](https://www.diycommitteeguide.org/) by [Volunteer Now](https://www.volunteernow.co.uk/) and is re-used under a Creative Commons Attribution Non-Commercial Share Alike 2.0. `reformat to reference: https://www.diycommitteeguide.org/what-is-the-role-of-the-secretary/`
+
+The Secretary supports the WG / DG in ensuring there is transparent documentation of the decisions made by the group.
+They are also accountable for tracking communications to broader *Turing Way* community from the WG, although they do not have to take on these responsibilities individually.
+
+### Responsibilities
+
+There are three dimensions of the Secretary's responsibilities:
+
+#### 1. Ensuring meetings are effectively organised and minuted
+
+The Secretary works closely with the Chair to plan meetings.
+They should give WG members sufficient time to add topics to the meeting agenda and circulate the agenda in advance of the meeting.
+
+We don't require working groups / delivery groups to take formal minutes of the meetings but we do ask that they capture and archive notes for two purposes: 
+- 1: to ensure members of the broader *Turing Way* community are able to participate in the activities of the WG / DG, including members of the WG / DG who are not able to attend synchronous meetings, and 
+- 2: to meet our guiding principle of transparent decision making. `format as reference`
+
+The Secretary keeps track of actions that have been identified and report to the WG / DG how they are progressing.
+The reporting is valuable to the Chair in their representation of the WG / DG at the constitutional decision making level through their membership of the Steering Council.
+The documentation also supports the WG / DG in keeping focused on their shared vision and goals.
+
+#### 2. Document membership of the WG and track acknowledgement of their work
+
+In *The Turing Way* we put acknowledgement of individuals' efforts and labour at the centre of our mission.
+The Secretary is asked to ensure that there is an up to date record of membership of the WG / DG, including alumni members.
+They are also asked to mentor members to regularly capture their efforts in the Contributors Record `reformat to reference: https://book.the-turing-way.org/afterword/contributors-record` of the book.
+
+#### 3. Track communication and correspondence
+
+As a combination of the two responsibilities above we ask that the Secretary keeps track of outputs and communications from the WG / DG.
+Their role is to ensure that the activities are not happening in a silo - meeting our guiding principle of reciprocity `format to reference` - and that the WG / DG members are appropriately celebrated for their efforts.
```

Comment:
```suggestion
Their role is to ensure that the activities are not happening in a silo - meeting our guiding principle of reciprocity `format to reference` - and that the Working / Delivery Group members are appropriately celebrated for their efforts.
```

I will note that our guiding principles document does not specifically reference reciprocity at the moment so we need to update